src/scss/settings/_variables.scss

```diff
@@ -118,9 +118,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
-$icon-path                 : '#{$img-path}/icons';
-$font-path                 : '/assets/dist/fonts';
+$img-path                  : 'fozzie/dist/img' !default; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
Might be worth catching up together on this one to go through how this works as I’m getting a little confused with the directory structure of the imported assets.

Probably easier to see it in action 👀